recipes-core/packagegroups/packagegroup-ni-internal-deps.bb

```diff
@@ -53,3 +53,10 @@ RDEPENDS:${PN}:append:x64 = "\
 RDEPENDS:${PN} += "\
 	libxml-parser-perl \
 "
+
+# Required for a mobilize step that installs a specific Python version 
+# and requires building Python on the test system
+# Contact: daq.sw.ops@ni.com
```

Comment:
Is there an actual engineer who can speak to the nature of this dependency?

Reply:
Yes, I can put the tech lead for that team instead of their group email. They indicated that they preferred this if possible though. Do I need to switch it to a specific person?

Reply:
Fixed.